linux_os/guide/system/software/disk_partitioning/encrypt_partitions/oval/shared.xml

```diff
@@ -0,0 +1,50 @@
+<def-group>
+    <definition class="compliance" id="{{{ rule_id }}}" version="1">
+        {{{ oval_metadata("Verify all partitions are encrypted except /boot /boot/efi",
+                            affected_platforms=["multi_platform_ol"]) }}}
```

Comment:
After the recent changes in Jinja macros, you need to pass `rule_title`  to `oval_metadata` explicitly. Otherwise, the generated OVAL will contain a void title. See the current output:

```
<oval-def:title>None</oval-def:title>
```

```suggestion
                            affected_platforms=["multi_platform_ol"], rule_title=rule_title) }}}
```